config/initializers/inflections.rb

```diff
@@ -10,7 +10,7 @@
 #   inflect.uncountable %w( fish sheep )
 # end
 ActiveSupport::Inflector.inflections(:en) do |inflect|
-  inflect.uncountable %w( feedback )
+  inflect.uncountable %w(feedback)
```

Comment:
This is inconsistent with `%w[string]` that we use elsewhere. We should probably use one or the other, not a mix of both.

Reply:
Hmm, I personally use `%w()` rather than square-brackets.

Reply:
That's fine, I don't mind which one, to be honest. I just think it should be consistent :)